akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/BehaviorTestKit.scala

```diff
@@ -4,24 +4,29 @@
 
 package akka.actor.testkit.typed.scaladsl
 
-import java.util.concurrent.ThreadLocalRandom
-
-import scala.collection.immutable
-import scala.reflect.ClassTag
-
+import akka.actor.testkit.typed.internal.{ ActorSystemStub, BehaviorTestKitImpl }
 import akka.actor.testkit.typed.{ CapturedLogEvent, Effect }
-import akka.actor.testkit.typed.internal.BehaviorTestKitImpl
-import akka.actor.typed.{ ActorRef, Behavior, Signal, TypedActorContext }
 import akka.actor.typed.receptionist.Receptionist
+import akka.actor.typed.{ ActorRef, Behavior, Signal, TypedActorContext }
 import akka.annotation.{ ApiMayChange, DoNotInherit }
+import com.typesafe.config.Config
+
+import java.util.concurrent.ThreadLocalRandom
+import scala.collection.immutable
+import scala.reflect.ClassTag
 
 @ApiMayChange
 object BehaviorTestKit {
-  import akka.actor.testkit.typed.scaladsl.TestInbox.address
 
-  def apply[T](initialBehavior: Behavior[T], name: String): BehaviorTestKit[T] = {
+  val ApplicationTestConfig: Config = ActorTestKit.ApplicationTestConfig
```

Comment:
ok, so this is only a way to expose this to the user if they want to use `application-test.conf` and pass that in as config to `BehaviorTestKit.apply`.

I would be open to not keep that backwards compatibility of using defaultReference as the default and instead align with ActorTestKit and use ApplicationTestConfig as the default here as well.

However, I'm fine with as it's implemented here as well.

Reply:
I'd also be happy to change the default behaviour as I find it faulty, however @johanandren wanted to avoid breaking backward compatibility in a minor version (Johan, please correct me if I'm mis-quoting you here)

Reply:
Yeah, I was thinking it might break existing user tests depending on config in their `application.conf` which would be unpleasant in a patch release. But maybe that is ok/and or unlikely.

Reply:
I think your previous comment that such unit test should work with a small standalone config, or the default as only reference.conf

Let's keep it like that.